src/lightning_app/utilities/scheduler.py

```diff
@@ -15,7 +15,7 @@ class SchedulerThread(threading.Thread):
     def __init__(self, app) -> None:
         super().__init__(daemon=True)
         self._exit_event = threading.Event()
-        self._sleep_time = 0.5
+        self._sleep_time = 0.05
```

Comment:
This puts extra pressure on the flow. I think it's ok to have a scheduler with a granularity in the order of magnitude of a second.
Maybe we can have a flag that shortens this during testing to speed it up if that is the need?

Reply:
Done !